src/pyhf/pdf.py

```diff
@@ -429,6 +429,8 @@ def set_poi(self, name):
         """
         Set the model parameter of interest to be model parameter ``name``.
 
+        If ``name`` is ``None``, this will reset the parameter of interest.
```

Comment:
"Reset" could maybe be misunderstood as "set back to what was originally in the spec from which the model was built"? I cannot think of another word to prevent this though.

Reply:
"unset" sounds very clear now, thanks!